model/datatable/DeliveriesMonitorDatatable.php

```diff
@@ -135,6 +136,6 @@ public function jsonSerialize()
      */
     private function getDeliveryHelperService()
     {
-        return $this->getServiceLocator()->get(DeliveryExecutionList::class);
+        return $this->getServiceLocator()->get(DeliveryExecutionListInterface::SERVICE_ID);
```

Comment:
`DeliveryExecutionList::class` or `DeliveryExecutionListInterface::class` should still work just fine.
Let's try minimizing the deprecated `SERVICE_ID` usage.

Reply:
you are right, i missed that magic https://github.com/oat-sa/generis/blob/master/common/oatbox/service/ServiceManager.php#L97